### ISSUE TITLE
[Bug] Mutation response `entry` returns null for non-authenticated users.

### ISSUE THREAD

**Issue by robmarshall** [harness-software/wp-graphql-gravity-forms#222]
I am working on https://github.com/robmarshall/gatsby-plugin-gravity-forms and since the updates made within 1.10.0 have been unable to successfully submit a form.

The mutation looks like this:
```
mutation submitForm($databaseId: ID!, $fieldValues: [FormFieldValuesInput]) {
    submitGfForm(
      input: { databaseId: $databaseId, fieldValues: $fieldValues }
    ) {
      entry {
        id
      }
      errors {
        id
        message
      }
    }
  }
```

The payload looks like this:

```
{
    "operationName": "submitForm",
    "variables": {
        "databaseId": 1,
        "fieldValues": [
            {
                "id": 1,
                "value": "my-value-here"
            }
        ]
    },
    "query": "mutation submitForm($databaseId: ID!, $fieldValues: [FormFieldValuesInput]) {\n  submitGfForm(input: {databaseId: $databaseId, fieldValues: $fieldValues}) {\n    entry {\n      id\n      __typename\n    }\n    errors {\n      id\n      message\n      __typename\n    }\n    __typename\n  }\n}\n"
}
```

But am getting the following response:
Type loader is expected to return a callable or valid type \"FormFieldValuesInput\", but it returned null"

Unsure where I am going wrong.

edit: formatting.

**Comment by robmarshall** [comment]
That mutation (yours) works perfectly in GraphiQL, but no luck within the app. However still getting the same error within the app.

I am using Apollo to handle the app side GraphQl functionality. See here: https://github.com/robmarshall/gatsby-plugin-gravity-forms/blob/main/src/submitMutation.js

WPGraphQL for GF version: 0.10.2
WPGraphQL version: 1.6.12
WordPress version: 5.9
WPGatsby version: 2.3.2
GF Version: 2.5.16
PHP version: 8.0.12

**Comment by justlevine** [comment]
Okay, tracked this down to a permissions issue. As of v0.10+ mutations now use the same entry/draft entry `Model` as queries, which means by default only admin / _validated users who submitted the form_ [can view the entry data](https://github.com/harness-software/wp-graphql-gravity-forms/blob/2cdf330bd2d2df3ad06d30b2bbc3344a9a90c4cd/src/Model/SubmittedEntry.php#L42). If a user isnt logged in,  `entry` returns null. 

The fix will take until after the weekend, but in the interim ( ⚠for testing purposes only, not recommended live since it willl expose your entries publicly), you can add the following snippet to your WP `functions.php`, so it doesnt get in the way of your work on  `gatsby-plugin-gravity-forms`.

```
add_filter( 'graphql_gf_can_view_draft_entries', '__return_true' );
add_filter( 'graphql_gf_can_view_entries', '__return_true' );

**Comment by justlevine** [comment]
@robmarshall if you run into any other issues with upgrading the Gatsby plugin, feel free to hmu on the WPGraphQL slack :)

**Comment by justlevine** [comment]
Fix is live in [v0.10.3 ](https://github.com/harness-software/wp-graphql-gravity-forms/releases/tag/v0.10.3)
